packages/tree-select/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+	"compilerOptions": {
+		"target": "es5",
+		"module": "es2015",
+		"declaration": true,
+		"declarationDir": "types",
```

Comment:
This config produces type information, the other does not. Types only need to be produced once.